### DISCUSSION TITLE
How can we enlarge the QR code on Certificate?

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#468] [General Discussion]
After downloading the certificate the QR code is not getting scanned.
The QR code is blurred on the certificate and not getting scanned but when we enlarge the font or resolution it is working fine.
SO, CAN WE ENLARGE THE QR CODE ON THE CERTIFICATE? IF YES THEN HOW
**This is the certificate pdf and screenshot** 
[BASIC COURSE.pdf](https://github.com/Sunbird-Ed/Community/files/11271348/BASIC.COURSE.pdf)
![image](https://user-images.githubusercontent.com/109504832/233031610-f546fbed-1725-4344-83c9-374f326d3858.png)
[@vinukumar-vs](https://github.com/vinukumar-vs) [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Comment by ashokreddy1208** [comment]
@sachinb80 We are able to scan QR code available on certificate using Sunbird ED mobile app without any issue, please let us know which app is used by you to scan QR codes.

[@vinukumar-vs](https://github.com/vinukumar-vs) [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Reply by sachinb80** [reply]
@ashokreddy1208 Yes we are able to scan the QR on certificate using any app but the problem is we need to enlarge the font of QR code then it is working.
Normal font is not getting scanned that we download after completing the course.
I have provided the sample certificate **BASIC COURSE**. Please download it and try scanning it

**Reply by ashokreddy1208** [reply]
@sachinb80 I am using same PDF sample which is shared by you, I am able to scan the QR code with regular font size.  please let us know which app is used by you to scan QR codes.

[@vinukumar-vs](https://github.com/vinukumar-vs) [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Reply by sachinb80** [reply]
We are using normal scanning app from mobile and Sunbird saas app
@ashokreddy1208 Can please share the version of the app u are using

**Reply by ashokreddy1208** [reply]
@sachinb80 We've tried this scenario in the SunbirdED 5.2 and 5.1 releases and it is working fine
